guide/interactions/slash-commands.md

```diff
@@ -179,7 +179,39 @@ const data = new SlashCommandBuilder()
 			.setName('server')
 			.setDescription('Info about the server'));
 ```
+### Localizations
 
+Localized names and descriptions depending on the user's selected language.
+
+You can pick localizations from [this list](https://discord.com/developers/docs/reference#locales)
```

Comment:
This first sentence feels... plain or bland to me?

> The names and descriptions of slash commands can be localized to the user's selected language The accepted locales can be seen from [this list](https://discord.com/developers/docs/reference#locales).

This feels a bit better.